README.md

```diff
@@ -1,3 +1,38 @@
-# blockprotocol
+<div id="top"></div>
```

Comment:
GitHub supports https://github.com/blockprotocol/blockprotocol#readme if that’s what you’re after 👀